modules/drivers/googleanalytics/src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -164,11 +167,11 @@
 
 (defmethod parse-filter :and
   [[_ & clauses]]
-  (str/join ";" (filter some? (map parse-filter clauses))))
+  (str/join ";" (remove str/blank? (map parse-filter clauses))))
```

Comment:
Purely for my own edification, why replace `filter some?` with `remove str/blank?`?